recipe/conda_build_config.yaml

```diff
@@ -2,17 +2,26 @@ c_compiler_version:        # [linux]
   - 12                     # [linux]
 cxx_compiler_version:      # [linux]
   - 12                     # [linux]
-arm_variant_type:  # [aarch64]
-  - sbsa           # [aarch64]
-VCVER:
-  - 14.2
-VSYEAR:
-  - 2019
-CL_VERSION:
-  - 19.29.30139
-  
+arm_variant_type:  # [linux and aarch64]
+  - sbsa           # [linux and aarch64]
+  - tegra          # [linux and aarch64]
```

Comment:
As we will want a migrator for this addition, think we should look at how we factor this out into a `conda-forge-pinning` migrator. That will also help us keep this in sync when rolling it out across the CTK and then third-party packages